velox/common/file/CMakeLists.txt

```diff
@@ -20,6 +20,6 @@ target_link_libraries(
   PUBLIC velox_exception Folly::folly
   PRIVATE velox_common_base fmt::fmt glog::glog)
 
-if(${VELOX_BUILD_TESTING})
+if(${VELOX_BUILD_TESTING} OR ${VELOX_BUILD_TEST_UTILS})
```

Comment:
Typically if we have test utils on and build testing off, google test is disabled, so this might fail.